stable/mission-control/CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # JFrog Mission-Control Chart Changelog
 All changes to this chart will be documented in this file.
 
+## [4.4.0] - Aug 25, 2020
+* **IMPORTANT**
```

Comment:
Say it is breaking please @chukka

Reply:
@chukka just this, please. Indicate that you are changing the `internalHttpPort` to `internalPort` and that this is breaking

Reply:
Done